rero_ils/modules/acq_receipt_lines/jsonschemas/acq_receipt_lines/acq_receipt_line-v0.0.1.json

```diff
@@ -83,6 +84,24 @@
         }
       }
     },
+    "vat_rate": {
```

Comment:
I think it is `overkill` to have a `vat_rate` by `receipt_line`. if it is the same `vat_rate` for all the items, I do not see myself adding the same field many times. I hope the `UI` will be smart enough to avoid this case.